src/templates/Pool/NewPoolOperations/Form/Invest/index.tsx

```diff
@@ -120,6 +122,14 @@ const Invest = ({ typeAction, privateInvestors }: IInvestProps) => {
   const { txNotification, transactionErrors } = useTransaction()
   const router = useRouter()
   const { data: pool } = usePoolData({ id: router?.query.address as string })
+  const { data } = usePoolInfo({
+    id: pool?.id || '',
```

Comment:
I'd use nullish coalescing opperator for this one, just a suggestion, do not need to refactor this one if you don't feel like doing so.

```suggestion
    id: pool?.id ?? '',
```